ejercicio03/index.html

```diff
@@ -13,6 +13,20 @@ <h3>Trujillo Colón Uziel</h3>
     <h1>Alumnos</h1>
 
     <h1>Descripciones</h1>
-    
+    <h2>Alumnos</h2>
+ <ol>
+
+	<h3>
+<li><p><a href="#C1">ver a Luis</a></li></p>
+	</h3>
+  
```

Comment:
Corregir identacion
```suggestion
     <ol>
            <li> Item </li>
      </ol>
```

La etiqueta ol, abarca toda la descripción, y son cosas diferentes:
1.- Es la lista
2.- El codigo de la descripcion